app/javascript/packs/session-timeout-ping.js

```diff
@@ -0,0 +1,135 @@
+/**
+ * @typedef NewRelicAgent
+ *
+ * @prop {(name:string,attributes:object)=>void} addPageAction Log page action to New Relic.
+ */
+
+/**
+ * @typedef LoginGov
+ *
+ * @prop {(any)=>void} Modal
+ * @prop {(string)=>void} autoLogout
+ * @prop {(el:HTMLElement?,timeLeft:number,endTime:number,interval?:number)=>void} countdownTimer
+ */
+
+/**
+ * @typedef NewRelicGlobals
+ *
+ * @prop {NewRelicAgent=} newrelic New Relic agent.
+ */
+
+/**
+ * @typedef LoginGovGlobals
+ *
+ * @prop {LoginGov} LoginGov
+ */
+
+/**
+ * @typedef {typeof window & NewRelicGlobals & LoginGovGlobals} LoginGovGlobal
+ */
+
+const login = /** @type {LoginGovGlobal} */ (window).LoginGov;
+
+const warningEl = document.getElementById('session-timeout-cntnr');
+
+const defaultTime = '60000';
```

Comment:
I'm not sure if the value is meaningful, but would I be correct in guessing this may have already been converted to milliseconds? It's being treated below in parsing as if it were seconds.

```suggestion
const defaultTime = '60';
```

Reply:
Oh right, I forgot the * 1000 is already on the end